health-services/individual/src/main/java/org/egov/individual/Constants.java

```diff
@@ -10,4 +10,6 @@ public interface Constants {
     String GET_ADDRESS = "getAddress";
     String GET_IDENTIFIER_TYPE = "getIdentifierType";
     String GET_ID = "getId";
+
+    String get();
```

Comment:
Why is this method get() added in constants file? Please check in only test classes